Library/Formula/libffi.rb

```diff
@@ -29,7 +29,7 @@ def install
     system "./autogen.sh" if build.head?
     system "./configure", "--disable-debug", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
-    system "make", "install"
+    make "install"
```

Comment:
This formula doesn't depend on `gmake`, so it shouldn't use the `make` helper. It's only designed for packages that depend on GNU make on Tiger.